src/h.js

```diff
@@ -288,6 +288,14 @@ const
         return meta ? (_dataCache.rdToken = meta.getAttribute('content')) : undefined;
     },
 
+    getSimilityCustomer = () => {
```

Comment:
I think this method should live inside the `payment-vm` itself. I don't see we using it in any other component. Our helper lib seems too cluttered already.